HIP/hip-0000-schedule-service-system-contract.md

```diff
@@ -0,0 +1,108 @@
+---
+hip: 
+title: Schedule Service System Contract
+author: Nana Essilfie-Conduah (@nana-ec)
+working-group: Richard Bair (@rbair23), Jasper Potts (@jasperpotts)
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 2023-06-14
+discussions-to: 
+---
+
+## Abstract
+
+This proposal addresses the feature gap of a smart contracts ability to issue scheduled transactions via the HAPI scheduled transactions.
+
+Since smart contracts executions do not utilize the Hedera signature map they are unable to carry along the authorizations that the Hedera ledger uses to confirm an accounts participation and acknowledgment in a transaction. 
+
+To address this Smart Contracts could utilize the Hedera Schedule Service by submitting a scheduled transaction to which accounts can sign / authorize as an acceptance of the desired transaction. This flow provides as easy route for asynchronous coordination of transaction approval.
```

Comment:
```suggestion
To address this Smart Contracts could utilize the Hedera Schedule Service by submitting a scheduled transaction to which accounts can sign / authorize as an acceptance of the desired transaction. This flow provides as easy route for asynchronous coordination of transaction approval.
```
To address this, Smart Contracts could utilize the Hedera Schedule Service by submitting a scheduled transaction to which accounts can subsequently sign / authorize as an acceptance of the desired transaction. This flow provides an easy route for asynchronous coordination of transaction approval.

Reply:
Updated